x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_view/category_stopped_partitions.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useContext, useEffect, useState, useMemo, useCallback } from 'react';
+import { EuiBasicTable, EuiCallOut, EuiSpacer, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { from } from 'rxjs';
+import { switchMap, takeWhile, tap } from 'rxjs/operators';
+import { JobCreatorContext } from '../../../job_creator_context';
+import { CategorizationJobCreator } from '../../../../../common/job_creator';
+import { ml } from '../../../../../../../services/ml_api_service';
+import { extractErrorProperties } from '../../../../../../../../../common/util/errors';
+
+const NUMBER_OF_PREVIEW = 5;
+export const CategoryStoppedPartitions: FC = () => {
+  const { jobCreator: jc, resultsLoader } = useContext(JobCreatorContext);
+  const jobCreator = jc as CategorizationJobCreator;
+  const [tableRow, setTableRow] = useState<Array<{ partitionName: string }>>([]);
+  const [stoppedPartitionsError, setStoppedPartitionsError] = useState<string | undefined>();
+
+  // using ref so this Subscription instance is only intialized once
+
+  const columns = useMemo(
+    () => [
+      {
+        field: 'partitionName',
+        name: i18n.translate(
+          'xpack.ml.newJob.wizard.pickFieldsStep.stoppedPartitionsPreviewColumnName',
+          {
+            defaultMessage: 'Stopped partition names',
+          }
+        ),
+        render: (partition: any) => (
+          <EuiText size="s">
+            <code>{partition}</code>
+          </EuiText>
+        ),
+      },
+    ],
+    []
+  );
+
+  const loadCategoryStoppedPartitions = useCallback(async () => {
+    try {
+      const results = await ml.results.getCategoryStoppedPartitions([jobCreator.jobId]);
+
+      if (
+        results?.jobs !== undefined &&
+        !Array.isArray(results?.jobs) && // if jobs is object of jobId: [partitions]
+        Array.isArray(results?.jobs[jobCreator.jobId]) &&
+        results.jobs[jobCreator.jobId].length > 0
+      ) {
+        return results.jobs[jobCreator.jobId];
+      }
+      return undefined;
```

Comment:
no need to return `undefined`

Reply:
Removed here https://github.com/elastic/kibana/pull/75709/commits/cc447272b32bf8b45cd20a0a39151cd5a096a289